R/install.R

```diff
@@ -48,6 +52,9 @@
 #' @param cpp_options A list specifying any makefile flags/variables to be
 #'   written to the `make/local` file. For example, `list("CXX" = "clang++")`
 #'   will force the use of clang for compilation.
+#' @param check_toolchain A list specifying any makefile flags/variables to be
```

Comment:
Just noticed that the doc is wrong here. I think it's just a copy/paste error. check_toolchain should be T/F right?

Reply:
Indeed. A copy paste error. Fixed now. Thanks!